CoreWordCloud/src/au/gov/asd/tac/constellation/views/wordcloud/content/ContentAnalysisManager.java

```diff
@@ -0,0 +1,265 @@
+        }
+
+        final Map<Integer, Integer> elementToCluster = ContentPairwiseSimilarityServices.clusterSimilarElements(th, nGramParams);
+
+        if (nGramParams.getFollowUpChoice().equals(ContentAnalysisOptions.FollowUpChoice.CLUSTER)) {
+            ContentAnalysisGraphProcessing.recordClusters(graph, elementToCluster);
+        } else if (nGramParams.getFollowUpChoice().equals(ContentAnalysisOptions.FollowUpChoice.MAKE_SELECTIONS)) {
+            ContentAnalysisGraphProcessing.makeSelectionsFromClusters(graph, elementToCluster);
+        }
+    }
+
+    // TODO: Implement plugin class and plugin params, then test, then gui!
+    public void phrasiphyContent(final PhrasiphyContentParameters phrasiphyContentParams) {
+        phrasiphyContent(phrasiphyContentParams, null);
+    }
+
+    private List<String> processBackground(final File file) {
+        if (file == null) {
+            return Collections.emptyList();
+        }
+        final List<String> lines = new ArrayList<>();
+
+        try (final BufferedReader background = new BufferedReader(new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8.name()))) {
+            String line = "";
+            while (line != null) {
+                lines.add(line);
+                line = background.readLine();
+            }
+        } catch (final IOException ex) {
+            LOGGER.log(Level.SEVERE, ex.getLocalizedMessage());
+        }
+        return lines;
+    }
+
+    public void phrasiphyContent(final PhrasiphyContentParameters phrasiphyContentParams, final File background) {
+        final PhraseTokenHandler bgHandler;
+        final List<String> lines = processBackground(background);
+        if (lines != null) {
+            final ThreadAllocator bgAllocator = getStringListAllocator(lines);
+            bgHandler = new PhraseTokenHandler();
+            ContentTokenizingServices.createPhraseAnalysisTokenizingService(bgHandler, phrasiphyContentParams, bgAllocator);
+        } else {
+            bgHandler = null;
+        }
+
+        final ThreadAllocator allocator = getGraphElementThreadAllocator();
+        final PhraseTokenHandler handler = new PhraseTokenHandler();
+        ContentTokenizingServices.createPhraseAnalysisTokenizingService(handler, phrasiphyContentParams, allocator);
+
+        final Future<?> f = PluginExecution.withPlugin(new SimpleEditPlugin("Display Word Cloud") {
+            @Override
+            public void edit(final GraphWriteMethods graph, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException {
+                int cloudAttr = graph.getAttribute(GraphElementType.META, WordCloud.WORD_CLOUD_ATTR);
+                final WordCloud wordCloud;
+                if (cloudAttr == Graph.NOT_FOUND) {
+                    cloudAttr = graph.addAttribute(GraphElementType.META, WordCloudAttributeDescription.ATTRIBUTE_NAME, WordCloud.WORD_CLOUD_ATTR, WordCloud.WORD_CLOUD_DESCR, null, null);
```

Comment:
Should a WordCloudConcept be added for this attribute (similar to other views)? Can also then just ensure this